pywinauto/xml_helpers.py

```diff
@@ -40,7 +40,15 @@
 import six
 import ctypes
 import re
-import bz2, base64
+import bz2
+
+try:
+    from base64 import encodestring
+    from base64 import decodestring
+except ImportError:
+    from base64 import encodebytes as encodestring
+    from base64 import decodebytes as decodestring

```

Comment:
Since `encodestring` and `decodestring` were deprecated starting with Python 3.9, I believe using a version bridge with `sys.version_info` is better than catching an `ImportError`.

Additionally, considering a future point when only Python 3.9+ will be supported, it would be better to use the existing names instead of continuing to rely on the deprecated ones.

https://docs.python.org/3/whatsnew/3.9.html#removed

```suggestion
import bz2
import sys

if sys.version_info >= (3, 9):
    from base64 import encodebytes
    from base64 import decodebytes
else:
    from base64 import encodestring as encodebytes
    from base64 import decodestring as decodebytes
```